arkouda/io.py

```diff
@@ -873,7 +876,7 @@ def to_parquet(
     # append or single column use the old logic
     if mode.lower() == "append" or len(pdarrays) == 1:
         for arr, name in zip(pdarrays, cast(List[str], datasetNames)):
-            arr.to_parquet(prefix_path=prefix_path, dataset=name, mode=mode, compressed=compressed)
+            arr.to_parquet(prefix_path=prefix_path, dataset=name, mode=mode, compressed=compression)
```

Comment:
should this be `compression=compression`? I'm a bit confused why it's not complaining about using a keyword arg that shouldn't exist anymore

Reply:
Updated.